curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/619b761916dac02643940022.md

```diff
@@ -11,6 +11,8 @@ A very common way to apply color to an element with CSS is with <dfn>hexadecimal
 
 Hex color values start with a `#` character and take six characters from 0-9 and A-F. The first pair of characters represent red, the second pair represent green, and the third pair represent blue. For example, `#4B5320`.
 
+The `A-F` letters that hex colors take is case insensitive. So, `a-f` still works fine. We'll be sticking with capital letters in our colored markers.
```

Comment:
"When typing hex color values, the A-F characters are case-insensitive."

Reply:
We should make the tests case-insensitive too (if we haven't already). It is kind of ridiculous to have to state this twice when we should just make our tests case-insensitive. If we did that, then we don't need to explain this. A lot of people are not going to know what "case-insensitive" means anyway.

Reply:
The test is case insensitive. So it works fine if learners type it in small letters. I've removed that whole line.